docs/getting-started/cli/subsembly-cli.md

```diff
@@ -0,0 +1,44 @@
+---
+description: cli overview and usage
+---
+
+# Install and Help
+
+## Overview
+
+**`subsembly`** is a cli tool that is used to ease the development of **`Subsembly`** runtimes. CLI tool contains following commands that help developers to easily compile and run **`Subsembly`** runtimes.
```

Comment:
Maybe we should reformat it to:
"Subsembly comes with a CLI tool to ease the development of `Subsembly` Runtimes...`